modules/packages/LinearAlgebra.chpl

```diff
@@ -2319,6 +2331,11 @@ proc kron(A: [?ADom] ?eltType, B: [?BDom] eltType) {
   return C;
 }
 
+proc expm(A: [],param useExactOneNorm = true) throws where isCSArr(A) {
```

Comment:
Add `pragma "no doc"`, and maybe say in the following documentation that `expm` accepts dense and CSR/CSC arrays.